### ISSUE TITLE
Kernel 'Conv2DBackpropFilter' not registered for backend 'wasm'

### ISSUE THREAD

**Issue by bjqsyq** [tensorflow/tfjs#8474]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow.js): I wrote custom code
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Windows 11 专业版
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow.js installed from (npm or script link): npm
- TensorFlow.js version (use command below): @tensorflow/tfjs-node@4.22.0, @tensorflow/tfjs-backend-wasm@4.22.0
- Browser version:
- Tensorflow.js Converter Version:@tensorflow/tfjs-converter@4.22

**Describe the current behavior**
tf.layers.conv2d are used in the model,Set the tf.setBackend('wasm')，Errors are reported during training：Error: Kernel 'Conv2DBackpropFilter' not registered for backend 'wasm'
**Describe the expected behavior**
No errors
**Standalone code to reproduce the issue**
<code>
import * as tf from '@tensorflow/tfjs-node';
import '@tensorflow/tfjs-backend-wasm';
createModel(){
    const model = tf.sequential();
    model.add(
      tf.layers.conv2d({
        inputShape: [130,6,1],
        kernelSize: 3,
        filters: 6,
        strides: 1,
        activation: 'relu',
        kernelInitializer: 'varianceScaling',
      }),
    );
    model.add(tf.layers.maxPooling2d({ poolSize: [2, 2], strides: [2, 2] }));
    model.add(
      tf.layers.conv2d({
        kernelSize: 1,
        filters: 2,
        strides: 1,
        activation: 'relu',
        kernelInitializer: 'varianceScaling',
      }),
    );
    model.add(tf.layers.maxPooling2d({ poolSize: [2, 2], strides: [2, 2] }));
    model.add(tf.layers.flatten());
    model.add(
      tf.layers.dense({
        units: 60,
        kernelRegularizer: tf.regularizers.l2({ l2: 0.001 }),
        kernelInitializer: 'varianceScaling',
        activation: 'relu',
      }),
    );
    model.add(
      tf.layers.dense({
        units:21,
        kernelInitializer: 'varianceScaling',
        activation: 'softmax',
      }),
    );
    const optimizer = tf.train.adam();
    model.compile({
      optimizer: optimizer,
      loss: 'categoricalCrossentropy',
      metrics: ['accuracy'],
    });
    return model;
  }
async modelFit(model,asyncData
): {
  this.setFitState(true);
  const data = tf.data
    .generator(asyncData)
    .shuffle(512)
    .batch(512);

  await model
    .fitDataset(data, {
      epochs: 10,
      callbacks: {
        onTrainBegin: async (logs) => {
          this.batchTmie = Date.now();
        },
        onBatchEnd: (batch, logs)=>{
          console.log(`batch = ${batch},times=${Date.now()`)
        },
        onEpochEnd: async (epoch, logs)=>{
          console.log(`Epoch ${epoch}:loss = ${logs.loss}`);
        },
      },
    })
    .then(async () => {
      await this.saveModel();
    })
    .catch((e) => {
         console.error(e
    });
}
const model=createModel()
modelFit(model, data);
</code>
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/CodePen/any notebook.

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by shmishra99** [comment]
Hi @bjqsyq , 

We appreciate your effort in highlighting this issue. `Conv2DBackpropFilter` isn't yet supported in the `WASM` backend. We've logged this as a feature request. Consider using `CPU, WebGL, or WebGPU` backends for now. i will provide updates on `WASM` backend support.  

 Thank You!!

**Comment by tripolskypetr** [comment]
💩

**Comment by nileshau-afk** [comment]
Hello @bjqsyq  ,

Thank you for bringing this to our attention and for providing a comprehensive overview. We appreciate the time you invested in creating this issue.

Currently, our internal development team is concentrated on other scheduled milestones, so we cannot prioritize this task immediately. However, we would love to see this move forward as a community-led effort.

If you’d like to submit a fix, please fork the repo and open a PR following the [TensorFlow Contribution Guidelines](https://www.tensorflow.org/community/contribute), [Tfjs Contributions](https://github.com/tensorflow/tfjs/blob/master/tfjs-vis/CONTRIBUTING.md#how-to-contribute) and be sure to link this issue in your PR to avoid duplicate work, kindly leave a comment here if you plan to take this on.

Thank you again for your interest and time.

**Comment by nileshau-afk** [comment]
Hello @bjqsyq  ,

Thank you for opening this issue. Since this issue has been open for a long time, we are closing this issue for now.
If this is still relevant and you would like to work on it, please feel free to reopen the issue or submit a pull request referencing it.

Thank you for your understanding.
